src/main/java/online/bookshop/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,69 @@
+package online.bookshop.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import online.bookshop.dto.cartitem.CartItemRequestDto;
+import online.bookshop.dto.cartitem.CartItemUpdateDto;
+import online.bookshop.dto.shoppingcart.ShoppingCartResponseDto;
+import online.bookshop.model.User;
+import online.bookshop.service.ShoppingCartService;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Shopping_cart management", description = "Endpoints for managing shoppingCart")
+@RestController
+@RequestMapping("/cart")
+@RequiredArgsConstructor
+public class ShoppingCartController {
+    private final ShoppingCartService shoppingCartService;
+
+    @Operation(summary = "Find a shoppingCart",
+            description = "Find a shoppingCart by userId into DB")
+    @GetMapping("/{id}")
```

Comment:
use authentication
```suggestion
    @GetMapping()
```
![image](https://github.com/OlexiiMelnik/book-shop/assets/20326858/304a59e9-639f-40a6-9cd9-6d7e5504999e)